specs/serving/knative-api-specification-1.0.md

```diff
@@ -362,12 +362,6 @@ code and configuration states (Revisions).
 Routes which are owned (controlled) by a Service SHOULD NOT be updated by
 developers; any changes will be reset by the Service controller.
 
-The table below details which operations must be made available to a developer
-accessing a Knative Route using a minimal profile. For any non-minimal profile,
-the POST, PUT, or DELETE operations MUST be enabled as a group. This ensures
-that the developer has the ability to control the complete lifecycle of the
-object from create through deletion.
```

Comment:
Why remove this verbiage?

Reply:
POST, PUT, or DELETE operations were marked as REQUIRED in tandem. The only remaining OPTIONAL operation is PATCH so it didn't seem necessary to keep this clause